src/consensus/aft/raft.h

```diff
@@ -2409,23 +2546,36 @@ namespace aft
 
       LOG_INFO_FMT(
         "Becoming candidate {}: {}", state->my_node_id, state->current_view);
+      LOG_DEBUG_FMT("Configurations: {}", configuration_tracker.to_string());
```

Comment:
This might be a bit too spammy, if there is not configuration changes in progress do we really need to print this log line?

Reply:
I need it, but nobody else does :-)